### PR TITLE
Avoid inline script for ExtJS label tooltips

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
We can't use `href="javascript:void(0)"` with a strict CSP so we need a new strategy for ExtJS-based tooltips. And we want to show tooltips on top of all open ExtJS windows/dialogs.

#### Changes
- Use newer tooltip approach for ExtJS labels
- Fix some HTML encoding problems

### CODE REVIEW

File: core/webapp/util.js [hunk 1]
```diff
@@ -47,13 +47,13 @@ function getHelpDiv()
     return _helpDiv;
 }
 
-function showHelpDivDelay(elem, titleText, bodyText, width, delay)
+function showHelpDivDelay(elem, titleText, bodyText, width, delay, zIndex)
 {
     // IE support
-    _showTimer = setTimeout(function() { showHelpDiv(elem, titleText, bodyText, width); }, delay ? delay : 400);
+    _showTimer = setTimeout(function() { showHelpDiv(elem, titleText, bodyText, width, zIndex); }, delay ? delay : 400);
```
Comment (labkey-adam) [inline core/webapp/util.js:53]: Still needed? "IE support"?

File: wiki/src/org/labkey/wiki/view/customizeWiki.jsp [hunk 2]
```diff
@@ -110,7 +110,7 @@ function disableSubmit()
     var btn = document.getElementById("btnSubmit");
     btn.disabled = true;
     btn.className = "labkey-disabled-button primary";
-    btn.href = "javascript:return false;";
+    btn.href = "#";
```
Comment (labkey-adam) [inline wiki/src/org/labkey/wiki/view/customizeWiki.jsp:113]: `#` vs. `return false` behave differently in some cases (refresh/navigate vs. no navigate). But probably fine.

**Review by labkey-adam — APPROVED** [review]
Consider updating "IE support" comment or code

File: wiki/src/org/labkey/wiki/view/customizeWiki.jsp (hunk 2 above)
Comment (labkey-jeckels) [reply wiki/src/org/labkey/wiki/view/customizeWiki.jsp:113]: I couldn't get the CSP to error on this one but decided to be conservative and keep an href

**Pull request merged**